src/finput.js

```diff
@@ -77,6 +77,9 @@ class Finput {
   get options() {
     return this._options;
   }
+  set options(options) {
+    this._options = options;
```

Comment:
This assumes that every option is set by the caller in `set options`. You'd most likely either wish to do a spread against the existing options `this._options = { ..._this.options, options }` (so the the set options call could only contain the values you want to change), or against the defaults (so that everything that ought to be set is defined)